src/activity-feed/ActivityFeedHeader.jsx

```diff
@@ -42,49 +43,54 @@ const HeaderActions = styled('div')`
   }
 `
 
-const Link = styled.a`
+const Link = styled('a')`
   margin-bottom: 0;
-`
-
-export default class ActivityFeedHeader extends React.Component {
-  static propTypes = {
-    totalActivities: PropTypes.number,
-    contentText: PropTypes.string,
-    contentLink: PropTypes.string,
+  margin-top: ${SPACING.SCALE_3};
+  @media (min-width: ${BREAKPOINTS.TABLET}) {
+    margin-left: ${SPACING.SCALE_3};
+    margin-top: 0;
   }
+`
 
-  static defaultProps = {
-    totalActivities: 0,
-    contentText: null,
-    contentLink: null,
-  }
+const ActivityFeedHeader = ({ totalActivities, buttons }) => {
+  const headerText = totalActivities
+    ? pluralize('activity', totalActivities, true)
+    : 'Activities'
 
-  render() {
-    const { totalActivities, contentText, contentLink } = this.props
-    const headerText = totalActivities
-      ? pluralize('activity', totalActivities, true)
-      : 'Activities'
+  return (
+    <HeaderSummary>
+      <HeaderCount>
+        <H2>{headerText}</H2>
+      </HeaderCount>
+      <HeaderActions>
+        {buttons?.map((button) => (
+          <Button
+            as={Link}
+            href={button.link}
+            buttonColour="#dee0e2"
+            buttonTextColour="#000"
+          >
+            {button.text}
+          </Button>
+        ))}
```

Comment:
Another (and more glexible) approach would be to render actions as plain JSX objects, e.g.
```
const ActionButton = ({ children, ...props }) =>
  <Button as={Link} buttonColour="#dee0e2" buttonTextColour="#000" {...props}>
    {children}
  </Button>


<ActivityFeed actions={
  <>
    <ActionButton href="...">Add interaction</Button>
    <ActionButton href="...">Add dad joke</Button>
  </>
}
```

And then style them using `HeaderActions`
```
const HeaderActions = styled('div')`
  a {
    margin-top: ${SPACING.SCALE_3};
    ...
  }
`
```

```suggestion
{actions && actons}
```

Reply:
@rafenden I've refactored this now to try this kind of approach.